### DISCUSSION TITLE
Upgrade cadence mysql db only?

### DISCUSSION THREAD

**Discussion by apolloturner** [uber/cadence#6372] [General]
Hello all,

I'm on cadence 0.23.2 and the database is aurora mysql 5.7. Can I upgrade the database to aurora mysql 8.0? Are there known issues?

**Comment by demirkayaender** [comment]
We have upgraded to MySql 8.0 last year, that should be fine. But your Cadence version is very low. You would need to update it to most recent version as well.

Since this upgrade involves a DB upgrade, I would recommend testing in a non-production environment for your own due diligence to verify.

**Comment by demirkayaender** [comment]
Closing since we talked over Slack. I shared there that we haven't done any code change while we were upgrading our MySQL version.
